Add way of working for leaving laptops unattended (#258)
MinBZK/ai-validation: docs/way-of-working/workplace-security.md

# Workplace Laptop Security

## Context

> When people leave their laptop open, unlocked, and unattended, this could be a big security breach, especially when you

Reviewer: ```suggestion
When people leave their laptop open, unlocked, and unattended, this could be a big security breach. Especially when you
```

have SSO on for multiple services.

## Decision

> Whenever a colleague will find your laptop open, unlocked, and unattended, they will message in the `random` Mattermost

Reviewer 2: I agree; you can, for example, open a document on the laptop and leave some kind of comment. This way, the person will also know that the laptop was left unattended without public bashing.

> Whenever a colleague will find your laptop open, unlocked, and unattended, they will message in the `random` Mattermost

Reviewer: ```suggestion
Whenever a colleague will find your laptop open, unlocked and unattended, they will message in the `random` Mattermost
```

> channel of the Rijks AI Validation team to bring cookies and/or cake to the office the next thursday as a way of

Reviewer: ```suggestion
channel of the Rijks AI Validation team to bring a small treat, e.g. cookies or cake to the office the next thursday as a way of
```
